components/table/nz-table.component.ts

```diff
@@ -269,6 +271,11 @@ export class NzTableComponent<T = any> implements OnInit, AfterViewInit, OnDestr
       this.fitScrollBar();
       this.setScrollPositionClassName();
     }
+    if (changes.nzData) {
+      if (this.platform.isBrowser) {
+        setTimeout(() => this.setScrollPositionClassName());
```

Comment:
This breaks the table for me for some reason:
![image](https://user-images.githubusercontent.com/36156020/66913404-3aa32900-f01d-11e9-853f-b3b1df315ad2.png)

8.3.0:
![image](https://user-images.githubusercontent.com/36156020/66913624-af766300-f01d-11e9-832f-aa2a25f6fae3.png)